_security/configuration/best-practices.md

```diff
@@ -0,0 +1,127 @@
+
+Before making changes with `securityadmin.sh`, always create a backup of the current configuration to prevent potential loss. For more detailed information, visit the [security admin configuration](https://opensearch.org/docs/latest/security/configuration/security-admin/) section in the documentation.
+
+### Benefits of using securityadmin.sh
+
+1. Consistency: By using securityadmin.sh, administrators can ensure consistency across security configurations within the cluster. This helps maintain a standardized and secure environment.
+2. Automation: securityadmin.sh enables automation of security configuration tasks, making it easier to deploy and manage security settings across multiple nodes or clusters.
+3. Version Control: Security configurations managed through securityadmin.sh can be version-controlled using standard version control systems like Git. This facilitates tracking changes, auditing, and reverting to previous configurations if necessary.
+
+To prevent configuration overrides, it's important to note that changes made through the OpenSearch Dashboards UI or using the OpenSearch API provide a convenient way to modify security settings. To ensure these updates are preserved when using the `securityadmin.sh` tool, first create a backup of the current configuration. This ensures all configurations are captured before uploading the modified configuration with `securityadmin.sh`.
+
+For more detailed information on using securityadmin.sh and managing security configurations in OpenSearch, refer to the following resources:
+- OpenSearch Security Documentation: [Applying changes to configuration files]({{site.url}}{{site.baseurl}}/security/configuration/security-admin/)
+- OpenSearch Security Documentation: [Modifying the YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/)
+
+## 9. Replace all default passwords
+
+When initializing OpenSearch with the demo configuration, many default passwords are provided in `internal_users.yml` for internal users such as `admin`, `kibanaserver`, `logstash`, and others.
+
+You should change the passwords for these users to strong, complex passwords either at startup or as soon as possible once the cluster is running. Creating password configurations is a straightforward procedure, especially using the scripts bundled with OpenSearch like `hash.sh` or `hash.bat`, located in the `plugin/OpenSearch security/tools` directory.
+
+The `kibanaserver` user is a crucial component that allows OpenSearch Dashboards to communicate with the OpenSearch cluster. By default, this user is preconfigured with a default password in the demo configuration. This should be replaced with a strong, unique password in the OpenSearch configuration, and the `opensearch_dashboards.yml` file should be updated to reflect this change.
+
+
+## 10. Stay informed and apply updates
```

Comment:
review-dog action seems to be mis-reading the word here.

Reply:
Yes indeed. Thanks.

Reply:
@kolchfa-aws: Very curious. Flagging this for you.